tide/wall/resources/wall.qrc

```diff
@@ -3,6 +3,7 @@
         <file>error.png</file>
     </qresource>
     <qresource prefix="/qml/wall">
+        <file>PowerTimer.qml</file>
```

Comment:
keep alphabetical order. Also, this isn't really a PowerTimer, more a PowerOffCountdown or such